### ISSUE TITLE
designsystem.digital.gov a11y: Scope

### ISSUE THREAD

**Issue by jAigret-Bix** [uswds/uswds-site#1093]
## Description
Related to PR #1092  a11y issues found re: `SCOPE`
Related to uswds/uswds/issues/3733

<details><summary>Design tokens - 1 axe violation</summary>

```bash
/design-tokens/ Found 1 aXe violations.
[
  {
    id: 'scope-attr-valid',
    impact: 'moderate',
    tags: [ 'cat.tables', 'best-practice' ],
    description: 'Ensures the scope attribute is used correctly on tables',
    help: 'scope attribute should be used correctly',
    helpUrl: 'https://dequeuniversity.com/rules/axe/3.5/scope-attr-valid?application=axeAPI',
    nodes: [
      [Object], [Object], [Object],
      [Object], [Object], [Object],
      [Object], [Object], [Object],
      [Object], [Object], [Object],
      [Object], [Object], [Object],
      [Object], [Object], [Object],
      [Object], [Object], [Object]
    ]
  }
]
```
</details>

<details><summary>Utilities - 1 axe violation</summary>

```bash
/utilities/ Found 1 aXe violations.
[
  {
    id: 'scope-attr-valid',
    impact: 'moderate',
    tags: [ 'cat.tables', 'best-practice' ],
    description: 'Ensures the scope attribute is used correctly on tables',
    help: 'scope attribute should be used correctly',
    helpUrl: 'https://dequeuniversity.com/rules/axe/3.5/scope-attr-valid?application=axeAPI',
    nodes: [
      [Object], [Object],
      [Object], [Object],
      [Object], [Object],
      [Object], [Object],
      [Object], [Object]
    ]
  }
]
```
</details>

⚠️ These issues will also appear on the individual Component pages once we start testing those. 

## Related Issues
- [ USWDS - Testing: Add ability to test subpages ie Component subpages. #1090](https://github.com/uswds/uswds-site/issues/1090)
- [ USWDS - Components: Create yml file for component variants. #1091](https://github.com/uswds/uswds-site/issues/1091)
